config.py

```diff
@@ -0,0 +1,46 @@
+import os
+
+basedir = os.path.abspath(os.path.dirname(__file__))
+
+
+class Config:
+    """
+    This holds the default configuration settings that will be inherited
+    for the software environment setup
+    """
+    SECRET_KEY = os.environ.get('SECRET_KEY') or 'check_point_rules'
+    SQLALCHEMY_COMMIT_ON_TEARDOWN = True
+    SQLALCHEMY_TRACK_MODIFICATIONS = False
+
+    @staticmethod
+    def init_app(app):
```

Comment:
I use the function in the application factor in the application factory in the create_app function in the __init__.py file in the app folder.